mix.exs

```diff
@@ -1,59 +1,66 @@
 defmodule UeberauthCognito.MixProject do
   use Mix.Project
 
+  @source_url "https://github.com/mbta/ueberauth_cognito"
   @version "0.3.0"
-  @url "https://github.com/mbta/ueberauth_cognito"
 
   def project do
     [
       app: :ueberauth_cognito,
+      name: "Ueberauth Cognito",
       version: @version,
       elixir: "~> 1.10",
       start_permanent: Mix.env() == :prod,
-      deps: deps(),
       test_coverage: [tool: LcovEx],
-      description: description(),
```

Comment:
It can be applied in both places (root or under package section), as in the end, ExDoc will [merge these values to a single map](https://www.kianmeng.org/2021/02/how-mix-hexbuild-process-and-use-elixir.html).

Reply:
That's interesting, I didn't realize that was possible. Nevertheless, I think I'd prefer to stick to whatever the published guidelines are. So for now, please do revert the `description` and `source_url` changes.

(That said, if you feel strongly about it, feel free to open a PR [to change the hex guidelines](https://github.com/hexpm/hexpm/blob/main/lib/hexpm_web/templates/docs/publish.html.md)!)

Reply:
Actually, I see that you enabled me to make edits, so I went ahead and made that change myself, and will merge this shortly.